docs/QuickStart-Introduction.md

```diff
@@ -0,0 +1,32 @@
+---
+id: quick-start-introduction
```

Comment:
What is the value of this page? This content should probably be on the front-page and getting started actually goes to the setup page?